Ansible 2

DenisV003 opened the pull request:
# Выполнено ДЗ №

 - [ *] Основное ДЗ
 - [ ] Задание со *

## В процессе сделано:
 - Выполнены все пункты


## Как запустить проект:
 - Например, запустить команду X в директории Y

## Как проверить работоспособность:
 - Например, перейти по ссылке http://localhost:8080

## PR checklist
 - [ *] Выставил label с номером домашнего задания
 - [ *] Выставил label с темой домашнего задания

Commit:
HW Ansible-1

Commit:
Add README.md

DenisV003 committed:
Update file

add space

DenisV003 committed:
Update clone.yml

add space

DenisV003 committed:
Update file

add space

DenisV003 committed:
Update file

add space

DenisV003 committed:
Update file

add space

DenisV003 committed:
Update file

add space

DenisV003 committed:
Update file

add space

DenisV003 committed:
Update file

add empty string

Commit:
HW ansible-2

Commit:
HW ansible-2 error correction

Commit:
HW ansible-2 key.json

Commit:
HW ansible-2 otus_key.json

Commit:
HW ansible-2 key.json new

Commit:
HW ansible-2 add new image

Commit:
HW ansible-2 add readme.md

Commit:
HW ansible-2 add space file

DenisV003 commented:
Добрый день. В автотесте почему то ругается на неизвестный ключ при validate packer. Хотя при запуске этих же команд на своей машине ошибок нет. 
dvoloshchik@voronezhoffice15  ~/Otus/VS/DenisV003_infra   ansible-2 ±  packer validate -var-file=packer/variables.json.example packer/db.json
The configuration is valid.

DenisV003 committed:
Merge branch 'main' into ansible-2

mrgreyves reviewed (changes requested):
Здравствуйте!)
Все верно
Было бы странно если на раннерах были ваши ключи /home/dvoloshchik/Otus/key.json
Добавьте фейковый ключ для пакера и в конфиге укажите пути для него

Commit:
HW ansible-2 add variables.json.example

Commit:
HW ansible-2 add variables.json.example

Commit:
HW ansible-2 new

Commit:
add .gitignore
Merge branch 'ansible-2' of github.com:Otus-DevOps-2023-01/DenisV003_infra into ansible-2

Commit:
HW ansible-2 add key.json

Commit:
HW ansible-2 add keynew.json

Commit:
HW ansible-2 add Otus_key.json

Commit:
HW ansible-2 add Otus_key.json

Commit:
HW ansible-2 add Otus_key.json

Commit:
HW ansible-2 add Otus_key.json

Commit:
HW ansible-2 add key.json

Commit:
HW ansible-2 add key.json

Commit:
HW ansible-2 add key.json

DenisV003 commented:
Это и есть фейковый ключ, который лежит в папке: /home/dvoloshchik/Otus/key.json. Ранее его же и использовал для выполнения ДЗ по packer, и автотесты приходили. Ниже добавил скриншот и содержимое. Пробовал другие пути, полный путь и через ./key.json. 
{
  "id": "ajevce7emn08l9c8nc76",
  "service_account_id": "ajevpu4ev0sv0jfqsdd8",
  "created_at": "2023-03-09T15:10:06.417174522Z",
  "key_algorithm": "RSA_2048",
  "public_key": "-----BEGIN PUBLIC KEY-----\nMIIBIjANBgkqhkiG9w0B\n-----END PUBLIC KEY-----\n",
  "private_key": "-----BEGIN PRIVATE KEY-----\nMIIEvAIBADANBgkqhki\n-----END PRIVATE KEY-----\n"
}
![image](https://user-images.githubusercontent.com/125120824/230341647-f3a27cfc-3b65-4033-a917-365c4cba5122.png)

Commit:
HW ansible-2 add key.json

Commit:
HW ansible-2 add key.json

DenisV003 commented:
Исправлено

mrgreyves reviewed (approved):
Здравствуйте!!)
Вот теперь все корректно =)
